garrysmod/gamemodes/terrortown/gamemode/cl_voice.lua

```diff
@@ -258,7 +258,7 @@ function RADIO:GetTargetType()
 
    local ent = trace.Entity
 
-   if ent:IsPlayer() then
+   if ent:IsPlayer()and  ent:IsTerror() then

```

Comment:
Good fix and I hate to nitpick, but please make the `and` have one space on either side.